### PR TITLE
Hotfix/fix model name

### PR DESCRIPTION
**Pull request by dpetzold**

### CODE REVIEW

File: cacheops/conf.py [hunk 1]
```diff
@@ -78,5 +78,4 @@ def model_profile(model):
     for guess in (app_model, '%s.*' % app, '*.*'):
         profile = model_profiles.get(guess)
         if profile:
-            profile['name'] = app_model
```
Comment (dpetzold) [inline cacheops/conf.py:81]: This function returns the cache profile for a model ie. the cache ttl. The bug was that with wildcard matches the model name would be incorrectly set to first model in the wildcard match which is not necessarily the model being looked up.

File: cacheops/query.py [hunk 2]
```diff
@@ -119,6 +119,7 @@ def _cacheprofile(self):
         profile = model_profile(self.model)
         if profile:
             self._cacheconf = profile.copy()
+            self._cacheconf['name'] = model_name(self.model)
```
Comment (dpetzold) [inline cacheops/query.py:122]: This is the correct place to set the model name as the wildcard matching has completed.

File: cacheops/query.py [hunk 3]
```diff
@@ -269,9 +270,9 @@ def iterator(self):
             cache_data, ttl = redis_client.get_with_ttl(cache_key)
             cache_read.send(
                 sender=self.model,
-                func=self._cacheprofile['name'],
+                func=self._cacheconf['name'],
```
Comment (Nesan) [inline cacheops/query.py:273]: Seems like we are storing the cache profile in two locations, one memoized in the _cacheprofile property, and another more explicitly in _cachconf.

**Review by Nesan — APPROVED** [review]
Changes look right after some explanation. (Thanks for the comments).

One confusing point, but looks shippable.

**Comment by Nesan** [comment]
Also, hurray for tests! Nice work @dpetzold

**Pull request closed without merging**